client/src/store/account.js

```diff
@@ -33,7 +33,13 @@ export default {
     // The Account Created mosaic.
     accountCreatedMosaics: [], // Wait for Rest team apply
     // The Account Transactions list.
-    accountTransactions: []
+    accountTransactions: [],
+
+    accountInfo2: {},
```

Comment:
what is the reason to have the suffix `2` ?

Reply:
I will change it to `formattedAccountInfo` . It is used in TableView, while `accountInfo` in another component